beluga-tunnel/src/lib.rs

```diff
@@ -133,9 +133,11 @@ impl Tunnel {
                                 service.connect(websocket_in, websocket_out, close_tx.clone()).await?;
                             }
                             Type::StreamReset => {
+                                debug!("stream reset isn't supported for now");
```

Comment:
Can we turn these into warnings, or do they show up way too often?
I want to make sure this doesn't go overlooked in case it's actually relevant.
```suggestion
                                debug!("stream reset isn't supported for now");
```